src/ssd1306-element-element.ts

```diff
@@ -1,12 +1,44 @@
 // Reference: https://cdn-learn.adafruit.com/assets/assets/000/036/494/original/lcds___displays_fabprint.png?1476374574
 import { customElement, html, LitElement, property, SVGTemplateResult } from 'lit-element';
 
-type NumTuple = [number, number];
+type Dimensions = [number, number];
 
 @customElement('wokwi-ssd1306-element')
 export class Ssd1306ElementElement extends LitElement {
-  @property() dimensions: NumTuple = [141, 108];
-  @property() bitmap: any; // size should be 128 x 64
+  @property() dimensions: Dimensions = [141, 108];
```

Comment:
It'd be better to have separate `width` / `height` properties, this way they can also be set via plain HTML when creating an instance of you component (also, it's pretty standard in HTML, e.g. the `<img>`, `<svg>` and `<canvas>` tags all follow this convention)

Reply:
done